user_operations.py

```
@@ -0,0 +1,139 @@
+            "Authorization": f"Bearer {token}",
+            "Content-Type": "application/json"
+        }
+        try:
+            response = requests.get(url, headers=headers)
+            if response.status_code != 200:
+                continue
+            user_data = response.json()
+            if not user_data.get("blocked", False):
+                unblocked.append(user_id)
+            # Update spinner
+            sys.stdout.write(f"\rChecking users... {spinner[spin_idx]} ({idx + 1}/{len(user_ids)})")
```

Ruff (0.11.9)

Undefined name `sys`

(F821)